dm/master/bootstrap_test.go

```diff
@@ -348,3 +353,10 @@ func (t *testMaster) TestSubtaskCfgsStagesV1Import(c *C) {
 	c.Assert(cfgs, HasLen, 0)
 	c.Assert(stages, HasLen, 0)
 }
+
+func checkAndAdjustSourceConfigMockFunc(ctx context.Context, cfg *config.SourceConfig) error {
```

Comment:
```suggestion
func checkAndAdjustSourceConfigMock(ctx context.Context, cfg *config.SourceConfig) error {
```

Reply:
the function has no Adjust in fact

Reply:
yes, i change this func name to `checkAndNoAdjustSourceConfigMock`, maybe we don't need to adjust source cfg in unit-test?